pyproject.toml

```diff
@@ -18,7 +18,7 @@ pylint = "^2.12.2"
 
 [tool.taskipy.tasks]
 test = "pytest --cov-report term-missing --cov-fail-under=70  --cov-branch --cov=hello tests/"
-lint = "pylint hello/ tests/"
+lint = "pylint tests/"
```

Comment:
Line 20 should read as: `test = "pytest --cov-report term-missing --cov-fail-under=70 --cov-branch --cov castanet tests/"`

Line 21 should read as `lint = "pylint castanet/ tests/"

Reply:
Please review what these lines should look like.